common/djangoapps/edxmako/makoloader.py

```diff
@@ -48,6 +48,8 @@ def load_template(self, template_name, template_dirs=None):
                                 module_directory=self.module_directory,
                                 input_encoding='utf-8',
                                 output_encoding='utf-8',
+                                default_filters=['decode.utf8'],
```

Comment:
@nedbat I can speak to this line looking good.  I'm not sure how to check that that other code is not needed.